clients/gtest/CMakeLists.txt

```diff
@@ -215,9 +240,11 @@ if (WIN32)
   )
   foreach( file_i ${third_party_dlls})
     add_custom_command( TARGET hipblas-test POST_BUILD COMMAND ${CMAKE_COMMAND} ARGS -E copy ${file_i} ${PROJECT_BINARY_DIR}/staging/ )
+    add_custom_command( TARGET hipblas_v2-test POST_BUILD COMMAND ${CMAKE_COMMAND} ARGS -E copy ${file_i} ${PROJECT_BINARY_DIR}/staging/ )
```

Comment:
Unless you expect only one target to be built you don't need to copy the same files twice.